shelter/README.md

```diff
@@ -28,11 +28,22 @@ git clone https://github.com/alibaba/inclavare-containers
 
 2. Prepare the dependence libs required by shelter
 
+### For EPID RA
+
 ```shell
 cd $WORKSPACE/inclavare-containers/shelter
 make
 ```
 
+### For DCAP RA
+
+Please refer to [this guide](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/README.md) to install DCAP. Note: If your platform is pre-product SGX platform (SBX), please follow this guide to resolve the quote verification problem on SBX platforms: https://github.com/alibaba/inclavare-containers/blob/master/hack/use-sbx-platform/README.md.
```

Comment:
`this guide` appeared twice here, You should revise both of them in markdown code style([this guide](https://github.com/alibaba/inclavare-containers/blob/master/hack/use-sbx-platform/README.md)), thanke.

Reply:
I quote this section from ra-tls README here:
https://github.com/alibaba/inclavare-containers/tree/master/ra-tls which follow Qianyue's suggestion.
I think we use different Markdown to separate them: one is quoted by IC from external source, one is quoted from IC internal source.

Reply:
> I quote this section from ra-tls README here:
> https://github.com/alibaba/inclavare-containers/tree/master/ra-tls which follow Qianyue's suggestion.
> I think we use different Markdown to separate them: one is quoted by IC from external source, one is quoted from IC internal source.

Ok. No problem.

Reply:
thx!